demosplan/DemosPlanCoreBundle/Security/Authentication/Provider/ApiUserProvider.php

```diff
@@ -48,16 +48,16 @@ public function loadUserByUsername($login): UserInterface
      */
     public function refreshUser(UserInterface $user): UserInterface
     {
-        return $user;
+        return $this->loadUserByIdentifier($user->getLogin());
```

Comment:
As mentioned by ReviewDog: 
In order to ensure availability of `->getLogin()`, the parameter $user,  probably should be `Entities/UserInterface` instead of 
`User/UserInterface`.